src/database.ts

```diff
@@ -2924,13 +2932,16 @@ class Database extends common.GrpcServiceObject {
     options?: TimestampBounds
   ): PartialResultStream {
     const proxyStream: Transform = through.obj();
+    const span = getActiveOrNoopSpan();
 
     this.pool_.getSession((err, session) => {
       if (err) {
         proxyStream.destroy(err);
         return;
       }
 
+      span.addEvent('Using Session', {'session.id': session?.id});
```

Comment:
Can you add the same event for all the places, where we are calling `this.pool_.getSession` in this file ? 

Same goes for `No session available` event in case of "Sessionnotfounderror"

Reply:
Roger that and done, thank you!